### PR TITLE
feat: page template with register form - TMPLT-366

### PR DESCRIPTION
**Pull request by Reinier1982**
Page Template with a register form.

### CODE REVIEW

File: src/prefabs/pageWithRegisterForm.js [hunk 1]
```diff
@@ -0,0 +1,8601 @@
...
+                            ],
+                          },
+                        },
+                        {
+                          label: 'Adornment',
+                          key: 'adornmentIcon',
+                          value: 'Email',
+                          type: 'CUSTOM',
+                          configuration: {
+                            as: 'DROPDOWN',
```
Comment (jeroenreijs) [inline src/prefabs/pageWithRegisterForm.js:4650]: I expected the `iconConfiguration` to be be used here

File: src/prefabs/pageWithRegisterForm.js [hunk 2]
```diff
@@ -0,0 +1,8601 @@
...
+                            ],
+                          },
+                        },
+                        {
+                          label: 'Show password toggle',
+                          key: 'adornment',
+                          value: true,
+                          type: 'TOGGLE',
+                        },
+                        {
```
Comment (jeroenreijs) [inline src/prefabs/pageWithRegisterForm.js:6744]: I expected the `iconConfiguration` to be be used here

**Review by jeroenreijs — CHANGES_REQUESTED** [review]

File: src/prefabs/pageWithRegisterForm.js (hunk 1 above)
Comment (RobD-tech) [reply src/prefabs/pageWithRegisterForm.js]: Added `iconConfiguration`

File: src/prefabs/pageWithRegisterForm.js (hunk 2 above)
Comment (RobD-tech) [reply src/prefabs/pageWithRegisterForm.js]: Added `iconConfiguration`

**Review by jeroenreijs — APPROVED** [review]

**Pull request merged**